src/mongocrypt-marking.c

```diff
@@ -611,6 +612,53 @@ _mongocrypt_fle2_placeholder_to_find_ciphertext (
    return res;
 }
 
+static bool
+_mongocrypt_fle2_placeholder_to_FLE2UnindexedEncryptedValue (
+   _mongocrypt_key_broker_t *kb,
+   _mongocrypt_marking_t *marking,
+   _mongocrypt_ciphertext_t *ciphertext,
+   mongocrypt_status_t *status)
+{
+   _mongocrypt_buffer_t plaintext = {0};
+   mc_FLE2EncryptionPlaceholder_t *placeholder = &marking->fle2;
+   _mongocrypt_buffer_t user_key = {0};
+   bool res = false;
+
+   BSON_ASSERT (marking->type == MONGOCRYPT_MARKING_FLE2_ENCRYPTION);
+   BSON_ASSERT (placeholder->algorithm == MONGOCRYPT_FLE2_ALGORITHM_UNINDEXED);
+   _mongocrypt_ciphertext_init (ciphertext);
+   _mongocrypt_buffer_init (&plaintext);
```

Comment:
I think this `buffer_init` is redundant with the immediate `buffer_from_iter` on the next line.

Reply:
Removed.